src/main/cpp/io/IntakeIO.cpp

```diff
@@ -36,30 +38,37 @@ bool IntakeIO::ProcessIO() {
         sw_interface_->reset_encoders = false;
     }
 
-    // intake position encoder
+    // === inputs ===
+    // intake encoder
     OKC_CHECK(hw_interface_->intake_motor != nullptr);
-    
-   
     sw_interface_->intake_encoder = hw_interface_->intake_encoder->GetPosition();
 
+    // wrist encoder
+    OKC_CHECK(hw_interface_->wrist_encoder != nullptr);
+    sw_interface_->tilt_encoder = hw_interface_->wrist_encoder->GetAbsolutePosition() + offset_;
+
+    // === outputs ===
     hw_interface_->intake_motor->Set(sw_interface_->intake_power);
+    hw_interface_->wrist_motor->Set(sw_interface_->tilt_power);
   
 
-return true;
+    return true;
 }
+
 bool IntakeIO::UpdateIntakeConfig(IntakeConfig &config) {
     OKC_CHECK(hw_interface_ != nullptr);
+    
+    // TODO
+
     return true;
 }
-    // Get the configuration
     
 //HACK: this is a hack
```

Comment:
this doesn't seem like a hack?

Reply:
it's a hack because I think you're supposed to only call GetEncoder() on a sparkmax once, and then make all subsequent calls to that encoder object instead of going through the motors. We had issues earlier with that not working and causing errors.